src/App.jsx

```diff
@@ -1,12 +1,57 @@
-import './App.css';
+import React, { useState } from 'react';
+import axios from 'axios';
+import './App.css';  // Voeg je eigen styling toe of pas de standaardstijl aan
+
+// Helper functie om de kleur voor de regio te bepalen
+const getRegionColor = (region) => {
+    const regionColors = {
+        "Africa": "blue",
+        "Americas": "green",
+        "Asia": "red",
+        "Europe": "yellow",
+        "Oceania": "purple",
+    };
+
+    return regionColors[region] || "gray";  // Als regio onbekend is, geef een neutrale kleur terug
+};
 
 function App() {
+    const [countries, setCountries] = useState([]);
+    const [loading, setLoading] = useState(false);
+
+    const fetchCountries = async () => {
+        setLoading(true);
+        try {
+            const response = await axios.get('https://restcountries.com/v3.1/all');
+            const sortedCountries = response.data.sort((a, b) => a.population - b.population);
```

Comment:
Wees ervan bewust dat je met de sort methode de originele array aanpast. Vaak is het handig om een kloon te maken van de originele array om vervolgens op de gekloonde array de methodes op los te gooien. Zo blijft de originele array intact.

hoe kloon je een array? zo dus:

`const clonedArray = [...originalArray]`